codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSwiftDependency.kt

```diff
@@ -11,34 +11,38 @@ class AWSSwiftDependency {
         val AWS_SDK_IDENTITY = SwiftDependency(
             "AWSSDKIdentity",
             "main",
-            "0.1.0",
```

Comment:
All these dependencies below have been updated with their registry scope and package name so they can be rendered as dependencies in Package.swift correctly.